hugo.toml

```diff
@@ -202,7 +202,7 @@ enable = false
 RelPermalink= "communityhealthtoolkit.org"
 
 [related]
-  treshold = 1
```

Comment:
I think we should just delete this line - I presume it wasn't doing anything before because of the typo, so unless we have a reason to start applying it I think it makes more sense to delete it.

Reply:
Done, thanks.